mainwindow.cpp

```diff
@@ -1,10 +1,9 @@
-/*: # mainwindow.cpp ([source](../appskeleton/mainwindows.cpp))
- * mainwindow.cpp contains the implementations of everything your window does. */
+/* mainwindow.cpp contains the implementations of everything your window does. */
 /*: The next two includes are our own headers that define the interfaces for
  * our window class and the recording device */
-#include "mainwindow.h"
-#include "reader.h"
-/*: `ui_mainwindow.h` is automatically generated from `mainwindow.ui`.
+#include "mainwindow.hpp"
+#include "reader.hpp"
+/*: `ui_mainwindow.hpp` is automatically generated from `mainwindow.ui`.
```

Comment:
.hpp extension is inconsistent with the `#include "ui_mainwindow.h"` below.

Reply:
Good catch, I've changed it back and will push it with the next batch of changes.